android/engine/src/main/java/org/smartregister/fhircore/engine/cql/R4FhirModelResolverExt.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021 Ona Systems, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.smartregister.fhircore.engine.cql
+
+import java.lang.System.out
+import org.opencds.cqf.cql.engine.fhir.model.R4FhirModelResolver
+
+class R4FhirModelResolverExt : R4FhirModelResolver() {
+  override fun resolveType(typeName: String): Class<*> {
+    // FHIR has a bug that does not allow processing inner static classes
```

Comment:
do you know what library this error is in? Have we created an issue or otherwise communicated the error to the community? Can you please link to reference here, thank you!

Reply:
https://github.com/DBCG/cql_engine/issues/537###

Reply:
thanks! adding to list of issues for google sprint